Cargo.lock

```diff
@@ -2214,13 +2214,15 @@ dependencies = [
 [[package]]
 name = "parity-ui-dev"
 version = "1.9.0"
+source = "git+https://github.com/parity-js/shell.git?rev=c4ad493b86fd8dda0364881d9cff39c95e2ee73b#c4ad493b86fd8dda0364881d9cff39c95e2ee73b"
```

Comment:
This auto-gen line _looks_ funny.

Reply:
It's because you have set explicit `rev` in `Cargo.toml` file. Instead you could just set up github repo and let `Cargo.lock` maintain the revision.
But it's fine, might be a bit more pain in the ass to update though:
```
1. edit Cargo.toml
2. Run $ cargo update -p parity-ui-dev
```
vs
```
$ cargo update -p parity-ui-dev
```

Reply:
100%, bit more of a pain with the revs - this does feel safer since the shell will morph as we get closer to having it dapp-able.